ccpp/physics_namelists/input_GFS_v17_HR3_RRTMGP.nml

```diff
@@ -0,0 +1,171 @@
+  ialb         = 1
+  iems         = 1
+  iaer         = 5111
+  icliq_sw     = 2
+  ico2         = 2
+  isubc_sw     = 2
+  isubc_lw     = 2
+  isol         = 2
+  lwhtr        = .true.
+  swhtr        = .true.
+  cnvgwd       = .true.
+  shal_cnv     = .true.
+  cal_pre      = .false.
+  redrag       = .true.
+  dspheat      = .true.
+  hybedmf      = .false.
+  satmedmf     = .true.
+  isatmedmf    = 1
+  lheatstrg    = .true.
+  lseaspray    = .true.
+  random_clds  = .false.
+  trans_trac   = .true.
+  cnvcld       = .true.
+  imfshalcnv   = 2
+  imfdeepcnv   = 2
+  ras          = .false.
+  cdmbgwd      = 10.0,3.5,1.0,1.0
+  prslrd0      = 0.
+  ivegsrc      = 1
+  isot         = 1
+  lsoil        = 4
+  lsm          = 2
+  iopt_dveg    = 4
+  iopt_crs     = 2
+  iopt_btr     = 1
+  iopt_run     = 1
+  iopt_sfc     = 3
+  iopt_trs     = 2
+  iopt_frz     = 1
+  iopt_inf     = 1
+  iopt_rad     = 3
+  iopt_alb     = 1
+  iopt_snf     = 4
+  iopt_tbot    = 2
+  iopt_stc     = 3
+  debug        = .false.
+  nstf_name    = 2,1,0,0,0
+  nst_anl      = .true.
+  psautco      = 0.0008,0.0005
+  prautco      = 0.00015,0.00015
+  lgfdlmprad   = .false.
+  effr_in      = .true.
+  ldiag_ugwp   = .false.
+  fscav_aero   = "'*:0.0'"
+  do_sppt      = .false.
+  do_shum      = .false.
+  do_skeb      = .false.
+  do_RRTMGP          = .true.
+  doGP_cldoptics_LUT = .true.
+  doGP_lwscat        = .true.
+  active_gases = 'h2o_co2_o3_n2o_ch4_o2'
+  ngases = 6
+  rrtmgp_root = '../../ccpp/physics/physics/Radiation/RRTMGP/rte-rrtmgp/'
+  lw_file_gas = 'rrtmgp/data/rrtmgp-data-lw-g128-210809.nc'
+  lw_file_clouds = 'extensions/cloud_optics/rrtmgp-cloud-optics-coeffs-lw.nc'
+  sw_file_gas = 'rrtmgp/data/rrtmgp-data-sw-g112-210809.nc'
+  sw_file_clouds = 'extensions/cloud_optics/rrtmgp-cloud-optics-coeffs-reordered-sw.nc'
+  rrtmgp_nGptsSW = 112
+  rrtmgp_nGptsLW = 128
+  rrtmgp_nBandsLW = 16
+  rrtmgp_nBandsSW = 14
+  frac_grid    = .true.
+  cplchm       = .false.
+  cplflx       = .false.
+  cplice       = .false.
+  cplwav       = .false.
+  cplwav2atm   = .false.
+  do_ca        = .false.
+  ca_global    = .false.
+  ca_sgs       = .false.
+  nca          = 1
+  ncells       = 5
+  nlives       = 12
+  nseed        = 1
+  nfracseed    = 0.5
+  nthresh      = 18
+  ca_trigger   = .true.
+  nspinup      = 1
+  iseed_ca     = 12345
+  lndp_type    = 0
+  n_var_lndp   = 0
+  do_ccpp_suite_sim = .false.
+/
+
+&cires_ugwp_nml
+  knob_ugwp_solver   = 2
+  knob_ugwp_version  = 1
+  knob_ugwp_source   = 1,1,0,0
+  knob_ugwp_wvspec   = 1,25,25,25
+  knob_ugwp_azdir    = 2,4,4,4
+  knob_ugwp_stoch    = 0,0,0,0
+  knob_ugwp_effac    = 1,1,1,1
+  knob_ugwp_doaxyz   = 1
+  knob_ugwp_doheat   = 1
+  knob_ugwp_dokdis   = 2
+  knob_ugwp_ndx4lh   = 4
+  knob_ugwp_palaunch = 275.0e2
+  knob_ugwp_nslope   = 0
+  knob_ugwp_lzmax    = 15.750e3
+  knob_ugwp_lzmin    = 0.75e3
+  knob_ugwp_lzstar   = 2.0e3
+  knob_ugwp_taumin   = 0.25e-3
+  knob_ugwp_tauamp   = 1.5e-3
```

Comment:
0.5e-3 for C786 and 0.35e-3 for C1152

Reply:
I wasn't planning on changing the resolutions from the HR3 SDF to the HR3_RRTMGP SDF. So whatever resolution you decided on is fine by me.